frontend/packages/dev-console/src/components/topology/filters/TopologyFilterBar.tsx

```diff
@@ -90,18 +97,23 @@ const TopologyFilterBar: React.FC<TopologyFilterBarProps> = ({
             </>
           }
         >
-          <Button variant="link" className="odc-topology-filter-bar__info-icon" isInline>
+          <Button variant="link" className="odc-topology-filter-bar__info-icon">
             <InfoCircleIcon />
           </Button>
         </Popover>
       </ToolbarGroup>
+      <ToolbarGroup className="odc-topology-filter-bar__kiali-link">
+        {kialiLink && <ExternalLink href={kialiLink} text="Kiali" />}
+      </ToolbarGroup>
```

Comment:
I think you want to wrap the group - otherwise you'll have a childless container.
```suggestion
      {kialiLink && (
        <ToolbarGroup className="odc-topology-filter-bar__kiali-link">
          <ExternalLink href={kialiLink} text="Kiali" />
        </ToolbarGroup>
      )}
```